Update code.py

elmwoodelec opened the pull request:
Noticed that midnight was not handled properly for cases like "Five to Midnight" or "Twenty after Midnight".  Was seeing noon instead of midnight.

elmwoodelec committed:
Update code.py

Noticed that midnight was not handled properly for cases like "Five to Midnight" or "Twenty after Midnight".  Was seeing noon instead of midnight.

brentru commented:
@elmwoodelec  Thank you for submitting a Pull Request. Your code is currently not passing the linter due to an `Undefined variable` named  `MIDIGHT` (Output: https://travis-ci.com/adafruit/Adafruit_Learning_System_Guides/builds/120074471#L1171).

Once you fix that up, re-commit and we'll take a look!

elmwoodelec committed:
Clock was showing noon instead of midnight

Added a special case so clock shows "xxx TO MIDNIGHT" or "xxx PAST MIDNIGHT"  instead of "xxx TO NOON" or "xxx PAST MIDNIGHT".  Midnight was only working properly if the display time was exactly midnight.

elmwoodelec reviewed (commented):
Noticed that midnight was not handled properly for cases like "Five to Midnight" or "Twenty after Midnight". Was seeing noon instead of midnight.

brentru commented:
@elmwoodelec  This branch is out-of-date with master, you'll need to merge the latest changes from `master` into it.

@TheKitty  Could you take a look at this PR and review?

TheKitty commented:
Let me know when ready and I'll check

On Thu, Aug 8, 2019 at 9:22 AM Brent Rubell <notifications@github.com>
wrote:

> @elmwoodelec <https://github.com/elmwoodelec> This branch is out-of-date
> with master, you'll need to merge the latest changes from master into it.
>
> @TheKitty <https://github.com/TheKitty> Could you take a look at this PR
> and review?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/adafruit/Adafruit_Learning_System_Guides/pull/775?email_source=notifications&email_token=AAOSY4GAEG6MYIGMX5Z6B2LQDQMZNA5CNFSM4IF4TAB2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD33S3VA#issuecomment-519515604>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOSY4FCZZET26G5HTCOI6LQDQMZNANCNFSM4IF4TABQ>
> .
>

ladyada committed:
Merge branch 'master' into patch-1

ladyada committed:
Merge branch 'master' into patch-1

dhalbert reviewed (approved):
Belated merge - thanks!
